projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
+	<!-- refactoring activeProjectDetail: Infosection with key actors, timelines etc-->
+	<div class="rtProjectInfoSection">
+	<section class="factorWrapper">
+	<div class="section group factors">
+		<div class="col span_3_of_12">
+			<div class="circle"><h3>Key factors</h3></div>
+			<div class="text"><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p></div>
+		</div>
+		<div class="col span_3_of_12">
+			<div class="circle"><h3>Problem owner</h3></div>
+			<div class="text"><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p></div>
+		</div>
+		<div class="col span_3_of_12">
+			<div class="circle"><h3>Project owner</h3></div>
+			<div class="text"><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p></div>
+		</div>
+		<div class="col span_3_of_12">
+			<div class="circle"><h3>Key Contributers</h3></div>
+			<div class="text"><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p></div>
+		</div>
+	</div>
+	</section>
+	
+	<div class="section group problem_owner content">
+		<div class="col span_4_of_12 image"></div>
+		<div class="col span_8_of_12"><h1>Problem owner</h1><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nec lectus ullamcorper, mollis quam at, tristique dui. Duis eu leo vel lectus luctus pharetra sed ut quam. Donec augue sem</p></div>
```

Comment:
Lets use [Bootstrap column classes](http://getbootstrap.com/css/#grid) here, so we have less CSS to maintain.